src/app/loadout/selectors.ts

```diff
@@ -133,7 +133,8 @@ export const selectedLoadoutStoreSelector = createSelector(
   sortedStoresSelector,
   currentStoreSelector,
   (rootState: RootState) => rootState.loadouts.selectedLoadoutStoreId,
-  (stores, currentStore, selectedLoadoutStoreId): DimStore => {
+  (stores, currentStore, selectedLoadoutStoreId): DimStore | undefined => {
+    // stores[0] can be undefined it seems
     const defaultStore = currentStore || stores[0];
```

Comment:
@bhollis thoughts on this? I only noticed that this can happen by pulling its usage up to the search context. It seems that these changes made a few assumptions about our types fail which is interesting.

Reply:
Yes, it can be undefined if stores aren't loaded, but in practice we often pretend like things can't be undefined because they're never used in a place where stores aren't loaded, and it makes things easier to deal with.